uberduck_ml_dev/data/parse.py

```diff
@@ -0,0 +1,163 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/data.parse.ipynb (unless otherwise specified).
+
+__all__ = ['CACHE_LOCATION', 'STANDARD_MULTISPEAKER', 'STANDARD_SINGLESPEAKER', 'VCTK']
+
+# Cell
+import os
+import numpy as np
+import sqlite3
+import uuid
+from pathlib import PosixPath
+from tqdm import tqdm
+from pathlib import Path
+import os
+import pandas as pd
+import json
+
+CACHE_LOCATION = Path.home() / Path(".cache/uberduck/uberduck-ml-exp.db")
+STANDARD_MULTISPEAKER = "standard-multispeaker"
+STANDARD_SINGLESPEAKER = "standard-singlespeaker"
+VCTK = "vctk"
+
+
+def _log_filelists(folder, fmt, conn, dataset_name: str = None):
+    """
+    logs a filelist into the speaker database
+    """
+    if fmt == STANDARD_MULTISPEAKER:
+        _parse_ms(root=folder, dataset_name=dataset_name)
+    if fmt == STANDARD_SINGLESPEAKER:
+        _parse_ss(
+            conn=conn,
+            root=folder,
+            speaker_name=dataset_name,
+            dir_path=folder,
+            dataset_name=dataset_name,
+        )
+    if fmt == VCTK:
+        raise
+
+
+def _add_speaker_to_db(
+    filelist_path: str,
+    speaker_name: str,
+    speaker_id=None,
+    dir_path: str = None,
+    rel_path: str = None,
+    dataset_name: str = None,
+    conn=None,
+):
+    """
+    filelist: the path of the filelist being added
+    speaker_name: the name of the speaker
+    dir_path: the path of the data repository containing the filelist
+    rel_path: the path of the wavs within the repository
+    dataset_name: the name of the dataset
+    """
+    uuid_ = uuid.uuid4()
+    if conn is None:
+        conn = sqlite3.connect(str(CACHE_LOCATION_EXP))
+    cursor = conn.cursor()
+    cursor.execute(
+        "INSERT OR REPLACE INTO FILELISTS VALUES (?, ?, ?,?,?,?,?)",
```

Comment:
Let's use consistent spacing. 
```suggestion
        "INSERT OR REPLACE INTO FILELISTS VALUES (?, ?, ?, ?, ?, ?, ?)",
```